File: examples/standalone/pipelineYaml/snippets/java/fraud-simulate-transaction.java
String[] parts = input.split("-");
long index = Long.parseLong(parts[parts.length - 1]);
String userId = "USER_" + String.format("%03d", index % 1000);
double amount = Math.round((10.0 + Math.random() * 4990.0) * 100.0) / 100.0;
String[] locations = {"NYC", "LAX", "CHI", "HOU", "PHX"};

org.json.JSONObject tx = new org.json.JSONObject();
tx.put("userId", userId);
tx.put("amount", amount);
tx.put("location", locations[(int)(index % 5)]);
tx.put("timestamp", java.time.Instant.now().toString());
return tx.toString();

File: examples/standalone/pipelineYaml/snippets/python/fraud-simulate-transaction.py
import json
import random

parts = input.split("-")
index = int(parts[-1])
user_id = f"USER_{index % 1000:03d}"
amount = round(10.0 + random.random() * 4990.0, 2)
locations = ["NYC", "LAX", "CHI", "HOU", "PHX"]
return json.dumps({"userId": user_id, "amount": amount, "location": locations[index % 5]})
